### DISCUSSION TITLE
Fl_Mac_App_Menu

### DISCUSSION THREAD

**Discussion by akhudek** [fltk-rs/fltk-rs#1390] [Q&A] [answered]
Is there any way to access FL_Mac_App_Menu currently? Or alternatively, is there any other way to set custom application menu items? On Mac it's typical to have settings or preferences under the application menu rather then elsewhere.

**Comment by MoAlyousef** [comment]
Currently by using SysMenuBar, although it might be limited.
I can probably bind Fl_Mac_App_Menu in the coming few days and it should be available for 1.4.2.

**Reply by akhudek** [reply]
That would be amazing! Though I didn't see it exposed in the C bindings either. Hopefully it's not too hard. :D

**Reply by MoAlyousef** [reply, marked as answer]
Thankfully it wasn't hard as it's an empty struct composed of static values. MacAppMenu was added in master in the menu module, along with setters for the values.

**Reply by akhudek** [reply]
Wow, you are the best! Thanks!
